addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/NetworkBindingConstants.java

```diff
@@ -8,11 +8,12 @@
  */
 package org.openhab.binding.network;
 
-import java.util.Collections;
 import java.util.Set;
 
 import org.eclipse.smarthome.core.thing.ThingTypeUID;
 
+import com.google.common.collect.Sets;
```

Comment:
The following would be another way:

```java
new HashSet(Arrays.asList("1", "2", "3"))
```

It wasn't me giving the comment you are referring to. But I agree that we should avoid dragging in guava just for this. Indeed, many bindings are using it currently, the direction is towards avoiding it though. On devices with restricted resources, it's a pretty high price to pay just for saving a few characters when creating collections...